packages/components/src/Divider/Divider.tsx

```diff
@@ -14,14 +14,38 @@ export interface DividerProps {
   width?: string
   border?: string
   color?: string
+  children?: React.ReactNode
 }
 
 export const Divider = styled.div<DividerProps>`
+  ${({ children, border, color, theme, width }) =>
+    children
+      ? `
+  display: flex;
+  align-items: center;
+  width: ${width || '100%'};
+
+  :before, :after {
+    flex: 1;
+    content: '';
+    padding: 0 1px 1px;
+    background-color: ${color || theme.colors.grey200};
+    margin: 16px;
+  }
+  
+  :before {
+    margin-left: 0;
+  }
+
+  :after {
+    margin-right: 0;
+  }
+      `
```

Comment:
Basically what does this do if Divider has children? Could this be documented in Storybook 🤔

Reply:
Yep sure! I introduced children prop to the divider to make components like this below.  As it serves the same purpose of a divider component, I made the prop change in the divider component 

![image](https://github.com/user-attachments/assets/88f97a32-d497-4fdd-a77c-70b3d2ba1d57)